### PR TITLE
Interactive List

### PR DESCRIPTION
**Pull request by anderCM**
- Add a new JavaScript file and import it as a module:
  - it will contain methods related to the status updates (completed: true / false).
- Add event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
- Implement a function for the "Clear all completed" button (use filter() method).
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/modules/RemoveTask.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  checkBox.addEventListener('change', function select() {
+    if (this.checked) {
+      addedInputTask.classList.add('text-decoration-line-through');
+      taskToDelete.push(idTask);
+      return;
+    }
+    addedInputTask.classList.remove('text-decoration-line-through');
+    taskToDelete = taskToDelete.filter(((task) => idTask !== task));
+  });
+};
```
Comment (julie-ify) [inline src/modules/RemoveTask.js]: - [x] When the checkbox is checked the `completed` property of the item in the local storage should be updated to `true`. Kindly implement that. Right now, only a strike-through is added to the task, but the property is not updated in the local storage.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @anderCM,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/EditTasks.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+const EditTask = (idTask) => {
+  const idLi = document.getElementById(idTask);
+  const addedInputTask = idLi.querySelector('input[type=text]');
+  const iconContainer = document.getElementById(`icon-container-${idTask}`);
+  idLi.classList.add('edit-task');
+  addedInputTask.classList.add('edit-task');
+  addedInputTask.disabled = false;
+  addedInputTask.focus();
+  iconContainer.innerHTML = `<i id="remove-task-${idTask}" class="remove-icon fa-solid fa-trash-can"></i>`;
```
Comment (Qoosim) [inline src/modules/EditTasks.js:14]: - [OPTIONAL] I noticed that after editing a task and reload the page, the modified text goes back to it initial state. Kindly look into that and fix. :point_down: 

[edit-funct.webm](https://user-images.githubusercontent.com/26597432/213548389-67f2df42-b5ac-4aa2-8ca6-3affdad6a4d4.webm)

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @anderCM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file but not professional. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/RemoveTask.js (hunk 1 above)
Comment (anderCM) [reply src/modules/RemoveTask.js]: Thanks for your comment. I am going to implement it

**Pull request merged**